moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Box, Typography, Chip, Grid } from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  filtersList: {
+    paddingTop: theme.spacing(1),
+    marginRight: "12px",
+    display: "flex",
+    alignItems: "start",
+  },
+  filtersText: {
+    fontFamily: "Roboto",
+    fontSize: ".9rem",
+    color: theme.palette.text.secondary,
+  },
+  filtersSpan: {
+    fontWeight: 600,
+    textTransform: "uppercase",
+  },
+}));
+
+/**
+ * Create text to show advanced filters and logic applied in the UI
+ * @param {Object} filters - The current filters applied
+ * @param {Boolean} isOr - The current logic applied
+ * @returns {string} - The text to display
+ */
+const makeFilteredByText = (filters, isOr) => {
+  const filtersCount = Object.keys(filters).length;
+  let filteredByText = "Filtered by ";
+
+  /* Only show logic string if more than one filter applied */
+  if (filtersCount === 1) return filteredByText;
+
+  if (isOr) {
+    filteredByText += "any ";
+  } else {
+    filteredByText += "all ";
+  }
+
+  return filteredByText;
+};
+
+/**
+ * Renders filters applied in advanced search
+ * @param {Object} filtersConfig - The filters configuration for the current table
+ * @param {Array|Objects} filters - applied filters
+ * @param {Boolean} isOr - true if ANY filters are matched, false if ALL
+ * @return {JSX.Element}
+ * @constructor
+ */
+const FiltersChips = ({ filters, isOr, filtersConfig }) => {
+  const classes = useStyles();
+
+  const filtersApplied = filters.map((filter) => {
```

Comment:
since this function returns JSX, i kinda want it to be it's own component, or see the JSX bits moved down to the where the `<Chip>` is rendered, so that this variable can be renamed to `filterLabels`.

Reply:
thanks for catching this, John!! I agree that the JSX should be handled in the return of the component. @chiaberry - could you update so the array of data for the chips is created here, and move the JSX returned here into the label prop of the chip?